tests/fingerprints/map.py

```diff
@@ -87,7 +91,10 @@ def test_map_sparse_count_fingerprint(smallest_smiles_list, smallest_mols_list):
     map_fp = MAPFingerprint(
         variant="count",
         sparse=True,
-        verbose=0,
+        verbose={
+            "dynamic_ncols": False,
+            "disable": True,
+        },
```

Comment:
Why?

Reply:
To test some different parametrizations within the tests.

Reply:
Ok, but for the final version, leave the existing tests unchanged. This need tests covering this specific functionality anyway.

Reply:
Removed verbose dictionary these tests.